### DISCUSSION TITLE
Theme from Oqtane Marketplace

### DISCUSSION THREAD

**Discussion by pnicosia80** [oqtane/oqtane.framework#3206] [Q&A] [answered]
I state that my Oqtane portal is compiled from source.
I would like to install a theme from the marketplace into my Oqtane installation, for example Studio-Elf Rocket Fuel. I have to install it from nuget on which project? My module project? or in the oqtane server project? Alternatively can I just install the package directly in the administration of the oqtane portal?

Thanks in advance

**Comment by sbwalker** [comment, marked as answer]
1. Browse to your Oqtane installation
2. Login as a host user
3. Access the Control Panel (gear icon at top-right)
4. Select Admin Dashboard
5. Select Theme Management
6. Select Install Theme
7. Click Download on Studio-Elf Rocket Fuel theme (this will download the nuget package to a /Packages folder in your installation)
8. Restart your installation and the theme will be automatically installed and will be available to add to your sites/pages

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/29fda5de-622c-4bae-96e4-2a83f9497361)

**Reply by pnicosia80** [reply]
Thank you for your answer
